packages/core/src/redux/slices/localMedia.ts

```diff
@@ -375,52 +374,20 @@ export const doUpdateDeviceList = createAppAsyncThunk(
                 return { devices: newDevices };
             }
 
-            const { changedDevices } = getUpdatedDevices({
+            const { changedDevices, addedDevices } = getUpdatedDevices({
                 oldDevices,
                 newDevices,
-                currentAudioId: selectCurrentMicrophoneDeviceId(state),
-                currentVideoId: selectCurrentCameraDeviceId(state),
```

Comment:
hm, I think we we should keep these. Maybe get the values directly from the stream instead? `stream.lastAudioId` and `stream.lastVideoId`

Reply:
I tested this out and it seems like lastAudioId and lastVideoId don't exist on the stream? at least for me when I've tested, both are always undefined 🤔

Reply:
right, that makes sense. Let's keep this as it was then

Reply:
I've tested it out as it was:
`currentAudioId: selectCurrentMicrophoneDeviceId(state),
 currentVideoId: selectCurrentCameraDeviceId(state),`

And this causes the same bug as before 🙃

Reply:
[for visibility]: We had a sync chat about this - decided to merge it as it is now. Switching devices works, but we might need to revisit remembering of the _last_ used devices later. Will create a follow up ticket for that